lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -138,7 +138,7 @@ sub _job_get($) {
 
     my $job = schema->resultset("Jobs")->search($search, \%attrs)->first;
     return unless $job;
-    return $job->to_hash(assets => 1);
+    return $job->to_hash(assets => 1, deps => 1);
 }
```

Comment:
I'm afraid of this little change - in how many places do we really use $job_get any more? I would prefer if you avoid job_get in the API and pick the job directly from DB instead of going through DBUS. And there you can then call to_hash with all you want.